airflow/providers/ssh/hooks/ssh.py

```diff
@@ -131,9 +132,11 @@ def __init__(  # pylint: disable=too-many-statements
                     self.key_file = extra_options.get("key_file")
 
                 private_key = extra_options.get('private_key')
-                private_key_passphrase = extra_options.get('private_key_passphrase')
+                self.private_key_passphrase = extra_options.get('private_key_passphrase')
```

Comment:
Making this an instance variable has no effect -- changing it won't do anything as the value has already been used by the time anything else could set this.

Reply:
The idea was to use it in SFTPHook subclass, not to change it.

Reply:
Ah gotcha